.github/workflows/versions.yaml

```diff
@@ -33,11 +33,23 @@ jobs:
         commit-message: "[bot] Automated version update"
         title: "[bot] Automated version update"
         body: |
+          ## Description
+
           This is an automated version and jsonnet dependencies update performed from CI on behalf of @paulfantom.
 
           Configuration of the workflow is located in `.github/workflows/versions.yaml`
+
+          ## Type of change
+
+          - [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)
+
+          ## Changelog entry
+          
+          ```release-note
+
+          ```
         team-reviewers: kube-prometheus-reviewers
-        labels: kind/enhancement
+        # labels: kind/enhancement
```

Comment:
Let's remove this line completely instead of adding dead code.

Reply:
SGTM :+1: